samples/TSQL/cursor.sql

```diff
@@ -0,0 +1,12 @@
+DECLARE Employee_Cursor CURSOR FOR  
+SELECT EmployeeID, Title
+FROM AdventureWorks2012.HumanResources.Employee;
+OPEN Employee_Cursor;
+FETCH NEXT FROM Employee_Cursor;
+WHILE @@FETCH_STATUS = 0  
+   BEGIN
+    FETCH NEXT FROM Employee_Cursor;
+END;
+CLOSE Employee_Cursor;
+DEALLOCATE Employee_Cursor;  
+GO
```

Comment:
All 3 of these were taken from various subsections of this section of Microsoft documentation https://docs.microsoft.com/en-us/sql/t-sql/queries/queries?view=sql-server-2017

Reply:
We can't use these samples, even if GitHub is part of Microsoft now, as it's not clear what license these files are covered by.

Reply:
It looks like this is the license covered by those files. https://github.com/MicrosoftDocs/sql-docs/blob/live/LICENSE

Reply:
Good find. I see the code itself is covered by MIT - https://github.com/MicrosoftDocs/sql-docs/blob/live/LICENSE-CODE - so we're good. 🙇